rust/Cargo.toml

```diff
@@ -10,3 +10,8 @@ crate-type = ["cdylib", "staticlib"] # Android needs cdylib, whereas iOS staticl
 [dependencies]
 anyhow = { version = "1", features = ["backtrace"] }
 flutter_rust_bridge = "1.49.0"
+bip39 = "1.0.1"
```

Comment:
I suspect you can just the version integrated with `bdk` instead by enabling the relevant feature flag? as a bonus, it enables to nicely create descriptor wallets from it (so your code would get simpler too!)